tests/messages_tests/test_mixins.py

```diff
@@ -1,14 +1,22 @@
-from django.test import SimpleTestCase, override_settings
+from django.test import TestCase, override_settings
 from django.urls import reverse
 
-from .urls import ContactFormViewWithMsg
+from .models import SomeObject
+from .urls import ContactFormViewWithMsg, DeleteFormViewWithMsg
 
 
 @override_settings(ROOT_URLCONF='messages_tests.urls')
-class SuccessMessageMixinTests(SimpleTestCase):
+class SuccessMessageMixinTests(TestCase):
 
     def test_set_messages_success(self):
         author = {'name': 'John Doe', 'slug': 'success-msg'}
         add_url = reverse('add_success_msg')
         req = self.client.post(add_url, author)
         self.assertIn(ContactFormViewWithMsg.success_message % author, req.cookies['messages'].value)
+
+    def test_set_messages_success_on_delete(self):
+        object_to_delete = SomeObject.objects.create(name="MyObject")
+        delete_url = reverse('success_msg_on_delete', args=[object_to_delete.pk])
+        req = self.client.delete(delete_url)
```

Comment:
I think use `post` here — does it make any sense to ask for a success message on DELETE?

Reply:
Changed it to `post`.